Cargo.toml

```diff
@@ -103,10 +103,10 @@ nix = { version = "0.25.0", optional = true }
 
 x11rb = { version = "0.11.1", features = [
     "allow-unsafe-code",
-    "present",
-    "render",
+    # "present",
+    # "render",
     "randr",
-    "xfixes",
+    # "xfixes",
```

Comment:
FWIW, we do use the `render` feature (it is enabled by one of the other features, so removing it here won't result in it not being enabled).

I've got a PR anyway that changes this along with some code in the X11 backend, so probably best to not change these here.